### DISCUSSION TITLE
What is the main place to ask questions about milvus? is there a discord server or something?

### DISCUSSION THREAD

**Discussion by martinenkoEduard** [milvus-io/milvus#18135] [Q&A and General discussion]
What is the main place to ask questions about milvus? is there a discord server or something?

**Comment by yhmo** [comment]
We have a discussion workspace on Slack.com:  milvusio.slack.com

Or here,  [github](https://github.com/milvus-io/milvus). [Discussion](https://github.com/milvus-io/milvus/discussions/) panel is for general questions. For some bugs need to be investigated, we suggest create an issue on [Issues](https://github.com/milvus-io/milvus/issues) panel.

**Reply by vaalessi** [reply]
Is the discussion workspace open? I tried to join and it appears you need an invite.

**Reply by yhmo** [reply]
> Is the discussion workspace open? I tried to join and it appears you need an invite.

I suppose the workspace is public, just click the right bottom button on the home page of milvus:
![Screenshot from 2023-02-02 10-21-09](https://user-images.githubusercontent.com/2282099/216218717-8c678c79-69f7-4297-ad66-c91a9a42033e.png)

If you could not join the workspace, try this invite link:
https://join.slack.com/t/milvusio/shared_invite/zt-1olv9lfr5-gDT1epShNbNCt~7OqN8heQ
